usr/share/rear/format/USB/default/300_format_usb_disk.sh

```diff
@@ -166,19 +166,19 @@ if is_positive_integer $USB_BOOT_PART_SIZE ; then
     # Create a boot partition for the bootloader config/plugins/modules, the kernel and the ReaR recovery system initrd.
     # Round boot partition size to nearest block size to make the next partition (the data partition) also align to the block size:
     USB_BOOT_PART_SIZE=$(( ( USB_BOOT_PART_SIZE + ( USB_PARTITION_ALIGN_BLOCK_SIZE / 2 ) ) / USB_PARTITION_ALIGN_BLOCK_SIZE * USB_PARTITION_ALIGN_BLOCK_SIZE ))
-    LogPrint "Creating boot partition $RAW_USB_DEVICE$current_partition_number with size $USB_BOOT_PART_SIZE MiB aligned at $USB_PARTITION_ALIGN_BLOCK_SIZE MiB"
+    LogPrint "Creating boot partition $current_partition_number on $RAW_USB_DEVICE with size $USB_BOOT_PART_SIZE MiB aligned at $USB_PARTITION_ALIGN_BLOCK_SIZE MiB"
```

Comment:
Only a cosmetic nitpick:
Perhaps it looks better to omit the word "device"
also in the other user messages?
In the other messages you changed
```
$RAW_USB_DEVICE$current_partition_number
```
to
```
$current_partition_number on device $RAW_USB_DEVICE
```
but here you changed it to only
```
$current_partition_number on $RAW_USB_DEVICE
```
i.e. without the word "device".

I could not test how it actually looks on the terminal
but from plain looking at the changes I think
the shorter form (i.e. without the word "device")
is better because I think in general it is better to
leave out words that are not needed by the user
to understand what a message is about.


For example in this case when
RAW_USB_DEVICE is something like /dev/nvme0n1
```
Creating boot partition 2 on /dev/nvme0n1
```
it should be clear that /dev/nvme0n1 is a device
so the word "device" not needed by the user
to understand what the message tells.